tools/src/test/java/org/apache/kafka/tools/consumer/group/DescribeConsumerGroupTest.java

```diff
@@ -536,13 +703,15 @@ public void testDescribeWithConsumersWithoutAssignedPartitions(ClusterInstance c
                 List<String> cgcArgs = new ArrayList<>(Arrays.asList("--bootstrap-server", clusterInstance.bootstrapServers(), "--describe", "--group", group));
                 cgcArgs.addAll(describeType);
                 // run two consumers in the group consuming from a single-partition topic
-                try (AutoCloseable protocolConsumerGroupExecutor = consumerGroupClosable(groupProtocol, group, topic, Collections.emptyMap(), 2);
+                try (AutoCloseable protocolConsumerGroupExecutor = consumerGroupClosable(groupProtocol, group, Set.of(topic), Collections.emptyMap(), 2);
```

Comment:
nit: It looks like that there is an overload which accepts `topic`. Hence we don't need to change all of them to using `Set.of(topic)`.